src/routes/transactions/transactions-history.controller.spec.ts

```diff
@@ -770,7 +771,7 @@ describe('Transactions History Controller (Unit)', () => {
   it('Should limit the amount of nested transfers', async () => {
     const safe = safeBuilder().build();
     const chain = chainBuilder().build();
-    const maxNestedTransfers = app
+    const maxNestedTransfers: number = app
```

Comment:
Nit: we can use the generic of `get` and `getOrThrow` here.

Reply:
I agree, thanks. Changed in [dbc6503](https://github.com/safe-global/safe-client-gateway/pull/1533/commits/dbc6503ff69aca7d3acde6c5b05c48da6927a29d)